docs/protocol.md

```diff
@@ -590,6 +591,10 @@ Value is string "host:port".
 
 Send claim message to `host:port` when response is being sent.
 
+### Transport Header `cn` -- Caller Name
+
+Value is name of the service making the call.
```

Comment:
other information of this sort (such as process name and host_port) is sent in the init req/res, not as part of every call.  can we not do the same with the caller name?